tests/Endpoints/SearchTest.php

```diff
@@ -879,4 +879,17 @@ public function testSearchWithDistinctAttribute(): void
         self::assertArrayHasKey('title', $response['hits'][0]);
         self::assertCount(1, $response['hits']);
     }
+
+    public function testSearchWithLocales(): void
```

Comment:
tests for localized attributes are missing?

Reply:
Oops, yes, good catch; I forgot to push the file.
It's done now 👍